package.json

```diff
@@ -26,14 +26,16 @@
   "homepage": "https://github.com/parse-server-modules/parse-files-utils#readme",
   "dependencies": {
     "async": "^2.0.0",
+    "fs": "0.0.1-security",
```

Comment:
what's the purpose of this? fs is a standard node package and doesn't need to be included?

https://www.npmjs.com/package/fs